src/client.js

```diff
@@ -1942,6 +1942,25 @@ MatrixClient.prototype.getAccountData = function(eventType) {
     return this.store.getAccountData(eventType);
 };
 
+/**
+ * Get account data event of given type for the current user. This variant
+ * bypasses the local store and gets account data directly from the homeserver,
+ * which can be useful very early in startup before the initial sync.
+ * @param {string} eventType The event type
+ * @return {module:client.Promise} Resolves: The contents of the given account
+ * data event.
+ * @return {module:http-api.MatrixError} Rejects: with an error response.
+ */
+MatrixClient.prototype.getAccountDataFromServer = function(eventType) {
```

Comment:
why not a `cache=false` on `getAccountData` ooi?

Reply:
Mostly because the original is sync whereas this one is async.

Reply:
ah, yes. Good call.